### ISSUE TITLE
Transaction Report saved lost

### ISSUE THREAD

**Issue by gabriele-v** [moneymanagerex/moneymanagerex#4623]
MMEX version:
 - [x] 1.5.15
 - [ ] 1.6.x

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

Not sure if there is a workaround considering the big changes, but I've lost all my Transaction Report after upgrading to 1.5.15
If fix is not easy/mandatory, issue can be closed.

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

**Comment by whalley** [comment]
Yes, unfortunately, due to the changes the existing reports are lost. I think the new functionality that has been added outweighs this inconvenience.

**Comment by vomikan** [comment]
Technically, all settings are still in the database, but are not taken into account by the new version of the program. You can run version 1.5.14 and see these settings. But, in my opinion, it's easier to configure it again than to transfer it with a script or some other algorithm.

**Comment by arp2164** [comment]
I have same problem, also saved report are lost. 
creating new transaction gives all zeros no matter which field you check.

**Comment by whalley** [comment]
> creating new transaction gives all zeros no matter which field you check.

Can you clarify what you are doing here? By 'transaction' do you mean 'transaction' report? If so, can you give an example/screenshot of what you have included in your filter.

**Comment by arp2164** [comment]
I have saved transaction reports with previous version and
After updating to new version, I lost all my saved reports.
So I tried to create new transaction report and it shows all zeros. No
values reported for withdrawals or deposits . No matter which fields you
checked or unchecked, no amounts are shown.
No I don’t have screen shot.
Please fix this problem.
Thanks
Ashok

**Comment by whalley** [comment]
@arp2164 You will need to include more information as you are the first to report this and we have not encountered it before. If you uncheck everything do you get a complete report of everything?

To assist can you please include a screenshot of the filter and the resultant output please.

**Comment by arp2164** [comment]
I have already send the screen shot to Mark Whalley. If I do not check any items (all unchecked) it gives me a lot of data from many years of all accounts.

**Comment by whalley** [comment]
> I have already send the screen shot to Mark Whalley. 

I don't have any screenshots, where did you send them.

> If I do not check any items (all unchecked) it gives me a lot of data from many years of all accounts.

Sounds like it is working, that is the expected behaviour.

**Comment by arp2164** [comment]
I emailed and attached the screenshots
That’s no good to see many data previously you could customize accordingly
you wanted to see
There is a bug
I am on Mac OS Monterey
Let me know

**Comment by whalley** [comment]
@arp2164 Where did you send the mail? I don't have anything. Please send to GitHub@ipx.co.uk

**Comment by arp2164** [comment]
I forwarded to your new email address. Here’s is the screenshot, I hope you can see it, image is add as txt.
![26D7C9A7-19D9-4754-A208-C89A5BEF657C](https://user-images.githubusercontent.com/107713258/174439612-696de2ba-ae70-4808-b768-ba816ad7a3e7.jpeg)

**Comment by vomikan** [comment]
Is transactions has split?
See bug #4643

**Comment by arp2164** [comment]
No there is no split amounts

**Comment by gabriele-v** [comment]
@vomikan maybe it's the "include similar" on categories? How is it managed in the new interface?

**Comment by vomikan** [comment]
To see all similar category:
<html>
<body>


bills.* | All categories starting with 'bills'
-- | --

</body>
</html>

It is necessary to enter the name of the main category and add ```.*```
https://github.com/moneymanagerex/moneymanagerex/issues/4618

**Comment by vomikan** [comment]
For Windows I recomment to download the newest release:
https://ci.appveyor.com/project/moneymanagerex/moneymanagerex/builds/43900561/job/3plxjnvhmfb8vy7y/artifacts

**Comment by gabriele-v** [comment]
Hmmm we agreed that regex are only for advanced users and now it's required for such a basic feature? I think we would need an easier way for that and the other one which we already discussed for the regex

**Comment by vomikan** [comment]
It's a bug. If Period range is added then nothing representer in the report,

![image](https://user-images.githubusercontent.com/6836805/174446536-79243d2f-e08d-4946-ab0e-509313b258c5.png)

**Comment by arp2164** [comment]
Is there a updated Mac OS version available?

**Comment by vomikan** [comment]
> Is there a updated Mac OS version available?

@whalley could you help with this issue?

**Comment by whalley** [comment]
Have uploaded the latest Beta with the latest fixes.
https://mmex.ipx.co.uk/

**Comment by arp2164** [comment]
If all unchecked I see whole list of numbers covering many years data.
If I select particular account to see then it show all zeros, here’s the
screenshot.

**Comment by whalley** [comment]
@arp2164  You will need to upload the screenshot here: https://github.com/moneymanagerex/moneymanagerex/issues/4623

Can you also confirm the version of MMEX you are using as your last communication on this topic was almost 4 months ago.
